src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Planta.kt

```diff
@@ -1,39 +1,41 @@
 package ar.edu.unahur.obj2.semillasAlViento
 
-abstract class Planta(val anioObtencionSemilla: Int, var altura: Float) {
+abstract class Planta(val anioObtencionSemilla: Int, var altura: Double) {
   fun esFuerte() = this.horasDeSolQueTolera() > 10
 
-  fun parcelaTieneComplicaciones(parcela: Parcela) =
-    parcela.plantas.any { it.horasDeSolQueTolera() < parcela.horasSolPorDia }
-
   abstract fun horasDeSolQueTolera(): Int
   abstract fun daSemillas(): Boolean
 }
 
-class Menta(anioObtencionSemilla: Int, altura: Float) : Planta(anioObtencionSemilla, altura) {
+class Menta(anioObtencionSemilla: Int, altura: Double) : Planta(anioObtencionSemilla, altura) {
   override fun horasDeSolQueTolera() = 6
   override fun daSemillas() = this.esFuerte() || altura > 0.4
```

Comment:
Acá podrían haber usado un template method, para no repetir código, teniendo en cuenta que la condición de si `esFuerte()` siempre se repite. 

```suggestion
  open fun daSemillas() = this.esFuerte()
  
  // Y después podrían hacer en las subclases
  override fun daSemillas() = super.daSemillas() || altura > 0.4